@here/harp-mapview/lib/ClipPlanesEvaluator.ts

```diff
@@ -717,6 +712,12 @@ export class TiltViewClipPlanesEvaluator extends TopViewClipPlanesEvaluator {
         return viewRanges;
     }
 
+    private getDistanceToHorizon(height: number): number {
```

Comment:
Please add some typedoc:
* what height?
* be more precise about distance to horizon (distance from where to where)?

Reply:
I would go further, because it is actually calculates the distance from point P to the sphere tangent point T that lays on the line between P and T.
Maybe getTangentDistance()... I don't know, most importantly this need to assert when negative value is passed as height! Point below sea level - depression.